pagopa/io-app-design-system
# [IOPLT-144] Map Legacy buttons versions to the context attribute

# ⚠️ THIS PR DEPENDS ON #56

## Short description
This PR implements the handler to chenge UI representation of the button components to use the legacy view based on `IOExperimentalDesignContext`

## How to test
Check on the example app in the button section components changes on toggle value change


https://github.com/pagopa/io-app-design-system/assets/3959405/611bef78-4ff2-40d6-9978-62b623a9b4a3
